services/adminService.js

```diff
@@ -53,29 +56,67 @@ exports.deleteUsers = async (users, isForceDelete) => {
     if (!user) {
       throw new ApiError(404, 'User not found')
     }
-    const reservation = await Reservation.find({ user: id })
-    if (reservation.length > 0 && isForceDelete) {
+    const reservations = await Reservation.find({ user: id })
+    if (reservations.length > 0 && isForceDelete) {
       await Reservation.deleteMany({ user: id })
+      reservations.forEach(async ({ user, hotel }) => {
```

Comment:
we are taking reservations for one user. Imagine:
User has 10 rervations and he will receive 10 emails.
Maybe it would be better to send one mail with message: "Your all reservations have been removed."